sw/airborne/modules/loggers/sdlogger_spi_direct.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) Bart Slinger
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+/**
+ * @file "modules/loggers/sdlogger_spi_direct.h"
+ * @author Bart Slinger
+ * SPI SD Logger that saves pprzlog messages to SD Card.
+ */
+
+#ifndef SDLOGGER_SPI_H
+#define SDLOGGER_SPI_H
+
+#define SDLOGGER_BUFFER_SIZE 128
+
+#include "std.h"
+#include "mcu_periph/link_device.h"
+#include "subsystems/radio_control.h"
```

Comment:
only link_device and spi includes actually seem to be needed in this header.
Better to only include the reset in the .c file where they are actually needed.